es-design-system/pages/molecules/es-error-page.vue

```diff
@@ -0,0 +1,105 @@
+<!-- eslint-disable vuejs-accessibility/no-static-element-interactions -->
+<template>
+    <div>
+        <h1>
+            EsErrorPage component
+        </h1>
+        <p>
+            The Error Page component is intended to be used in error pages across nuxt apps.
+        </p>
+        <div class="my-450">
+            <h2>
```

Comment:
Probably one example is enough for the documentation page - when people look at the template for this page under "Documentation" they'll be able to see how it's customized with i.e. a prop.  You could add a line in the description up top though about the 4 error codes explicitly handled?